best-practices/restrict_image_registries/restrict_image_registries.yaml

```diff
@@ -9,11 +9,14 @@ metadata:
     policies.kyverno.io/minversion: 1.3.0
     policies.kyverno.io/subject: Pod
     policies.kyverno.io/description: >-
-      Images from unknown registries may not be scanned and secured. 
-      Requiring use of known registries helps reduce threat exposure.
+      Images from unknown, public registries can be of dubious quality and may not be
+      scanned and secured, representing some degree of risk. Requiring use of known, approved
+      registries helps reduce threat exposure by ensuring image pulls only come from them. This
+      sample validates that container images only originate from the registry `k8s.gcr.io` or
```

Comment:
should we use a non-public registry in the sample?

Reply:
Changing to fictitious examples.